modules/liblc3codec/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 if(CONFIG_LIBLC3CODEC)
 
 zephyr_library_named(liblc3codec)
-zephyr_library_compile_options(-O3 -ffast-math -Wno-array-bounds)
+zephyr_library_compile_options(-O3 -std=c11 -ffast-math -Wno-array-bounds)
```

Comment:
Why did you add this? I don't think it is required?

Reply:
C11 added `alignas` specifier https://en.cppreference.com/w/cpp/language/alignas used in liblc3codec. Without the `c11` option I get compilation errors:

```
/home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf_arm.h: In function 'arm_resample_8k_12k8':
/home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf_arm.h:124:34: error: expected expression before 'int32_t'
  124 |     static const int16_t alignas(int32_t) h[2*8*12] = {
      |                                  ^~~~~~~
/home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf_arm.h:124:26: error: expected identifier or '(' before ')' token
  124 |     static const int16_t alignas(int32_t) h[2*8*12] = {
      |                          ^~~~~~~
In file included from /home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf.c:23:
/home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf_arm.h:144:31: error: 'h' undeclared (first use in this function)
  144 |         8, (const int16x2_t *)h, hp50, (int16x2_t *)x, y, n);
      |                               ^
/home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf_arm.h:144:31: note: each undeclared identifier is reported only once for each function it appears in
In file included from /home/mariuszskamra/zephyr-sdk/arm-zephyr-eabi/arm-zephyr-eabi/sys-include/math.h:6,
                 from /home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/fastmath.h:27,
                 from /home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/common.h:27,
                 from /home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf.h:29,
                 from /home/mariuszskamra/zephyrproject/modules/lib/liblc3codec/src/ltpf.c:19:
```